backend/app/router.go

```diff
@@ -27,3 +32,54 @@ func GetRouter() *gin.Engine {
 func GetApiRouter() *gin.RouterGroup {
 	return apiRouter
 }
+
+// GenerateJWT will generate the JWT token
+func GenerateJWT(customKey string) (string, error) {
+	token := jwt.New(jwt.SigningMethodHS256)
+
+	claims := token.Claims.(jwt.MapClaims)
+
+	claims["authorized"] = true
+	claims["client"] = customKey
+	claims["exp"] = time.Now().Add(time.Minute * 30).Unix()
+
+	tokenString, err := token.SignedString(jwtSigningKey)
+
+	if err != nil {
+		fmt.Errorf("Something Went Wrong: %s", err.Error())
+		return "", err
+	}
+
+	return tokenString, nil
+}
+
+// IsAuthorized This function will check the authorization from a given token in Bearer
+func IsAuthorized(endpoint func(c *gin.Context)) gin.HandlerFunc {
+	fn := func(c *gin.Context) {
+
```

Comment:
This new line is unecessary. this will fail during linting.
Delete it

Reply:
Don't get it... what is unecessary ?
which lines ?